sonic-slave-stretch/Dockerfile

```diff
@@ -109,7 +114,6 @@ RUN apt-get update && apt-get install -y \
         dpatch \
         libdb-dev \
         iptables-dev \
-        swig \
```

Comment:
this will break the swss common build, can you add swig into the swss_common build depends.